content/en/docs/components/pipelines/v2/author-a-pipeline/pipelines.md

```diff
@@ -68,6 +68,55 @@ def my_pipeline():
         train_model(epochs=epochs)
 ```
 
+### dsl.Collected
+
+Use [`dsl.Collected`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.Collected) with `dsl.ParallelFor` to gather outputs from a parallel loop of tasks:
+
+```python
+from kfp import dsl
+
+@dsl.pipeline
+def my_pipeline():
+    with dsl.ParallelFor(
+        items=[1, 5, 10, 25],
+        parallelism=2
+    ) as epochs:
+        train_model_task = train_model(epochs=epochs)
+    max_accuracy(models=dsl.Collected(train_model_task.outputs['model']))
+```
+
+Downstream tasks may consume `dsl.Collected` outputs via an input annotated with a `List` of parameters or `List` of artifacts. For example, `max_accuracy` in the above example has the input `models` with type `List[Model]`:
```

Comment:
Suggestions:
- Replace "may" with "might".
- Add "a" after "or".
- Replace "above" with "preceding". Our style recommendation generally discourages using "above" to denote position in a document.
- Is it possible to have a sentence describing the next code sample?

Reply:
Updated.